draft/examples/reconciliation-query-batch/valid/example-full.json

```diff
@@ -7,11 +7,17 @@
       "properties": [
         {
           "pid": "professionOrOccupation",
-          "v": "Politik*"
+          "v": "Politik*",
```

Comment:
Lexical matching seems like a stretch here. I think it would be much more likely for the user to want any sub-class of a given profession.

Reply:
Yes, true, that is a bit odd. This example stems from the GND data that our service exposes, where we don't have a common entity for female and male politicians, so this is a way to cover both. However, that line is not actually added with this PR, it's only green because of the comma, this has been the example before (not to say this should not be changed, but maybe not in this PR, in particular I was happy that this actually provided another usage of `match_qualifier`, using `WildcardMatch`, thus formalizing something that the service was doing implicitly).

Reply:
"Things not strings." If this can't be fixed in the GND model, I'd rather see the reconciliation service work around it by introducing its own fake super type than supporting lexical wildcarding on properties.

But since this wasn't a change introduced by this pull request, I'm happy deferring fixing it.